src/FileSystem-Tests-Core/FileSystemTest.class.st

```diff
@@ -679,6 +679,23 @@ FileSystemTest >> testRootIsNotAFile [
 	self deny: (filesystem isFile: Path root)
 ]
 
+{ #category : #'tests-streams-compatibility' }
+FileSystemTest >> testTooManyDirectoryDescriptorOpened [
+	| opened testSize |
+
+	self halt.
+	testSize := 1025.
+	opened := OrderedCollection new:testSize.
+
+	self should: 
+		[1 to: testSize do: [ :x | opened add: ((File primOpendir: '/etc') at:3)]] 
```

Comment:
you can use `x timesRepeat: []` instead of `#to:do:`

Reply:
### agreed